stardog/files/stardog.properties

```diff
@@ -1 +1,4 @@
 watchdog.enabled=false
+metrics.enabled=true
```

Comment:
Is there a way to make this configurable? This might have a performance impact and not everyone will need metrics enabled.